app/controllers/admin/lyrics_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Admin::LyricsController < Admin::BaseController
+  before_action :load_lyric, except: %i(index)
```

Comment:
nếu chỉ có 1 action thì ko cần tạo thành symbol array em nhé
sửa như này he:
```suggestion
  before_action :load_lyric, except: :index
```